ptbcontrib/decorators/handlers.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+#
+# A library containing community-based extension for the python-telegram-bot library
+# Copyright (C) 2020-2024
+# The ptbcontrib developers
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import telegram.ext as tg
+from telegram import Update
+from telegram._utils.defaultvalue import DEFAULT_TRUE
+from telegram._utils.types import DVType
+from telegram.ext import filters as filters_module
+from telegram.ext._utils.types import FilterDataDict
+
+
+class CustomCommandHandler(tg.CommandHandler):
```

Comment:
"Custom" is not a very descriptive name, please choose a more speaking name. Moreover, please document your handler classes to some extend so that one may understand what they are good for